lib/commands/log.js

```diff
@@ -37,12 +37,10 @@ export const supportedLogTypes = {
      */
     getter: (self) => {
       self.ensureFeatureEnabled(GET_SERVER_LOGS_FEATURE);
-      const timestamp = Date.now();
       return self.log
         .unwrap()
-        .record.map((x) =>
-          toLogRecord(
-            timestamp,
+        .record.map((x) => toLogRecord(
+            /** @type {any} */ (x).timestamp ?? Date.now(),
```

Comment:
btw, for old deps, the `timestamp` could be undefined so `Date.now()`  will be used, correct? (just a confirmation)

Reply:
yes, this is done in order to keep the compatibility with older server versions